File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
class Solution {
    public String solution(String s) {
        int max = Integer.MIN_VALUE;
        int min = Integer.MAX_VALUE;
        for(String n : s.split(" ")){
            int i = Integer.parseInt(n);
            max = Math.max(max, i);
            min = Math.min(min, i);
        }
        
        return min + " " + max;
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.cpp
#include <string>
#include <vector>
#include <sstream>
#include <algorithm>

using namespace std;

string solution(string s) {
    stringstream ss(s);
    int num;
    ss >> num;
    
    int min_val = num;
    int max_val = num;
    
    while(ss >> num){
        min_val = min(min_val, num);
        max_val = max(max_val, num);
    }
    
    return to_string(min_val) + " " + to_string(max_val);
    
}